regression-tests/sparktkregtests/testcases/frames/frame_create_from_csv_test.py

```diff
@@ -0,0 +1,128 @@
+""" Tests import_csv functionality with varying parameters"""
+
+import sys
+import unittest
+from sparktkregtests.lib import sparktk_test
+import sparktk
+
+
+class FrameImportCSVTest(sparktk_test.SparkTKTestCase):
+
+    def setUp(self):
+        """Build frames to be exercised and establish known baselines"""
+        super(FrameImportCSVTest, self).setUp()
+        self.dataset = self.get_file("int_str_int.csv")
+        self.schema = [("num1", int), ("letter", str), ("num2", int)]
+        self.frame = self.context.frame.import_csv(self.dataset, schema=self.schema)
+
+    def test_frame_invalid_column(self):
+        """Tests retrieving an invalid column errors."""
+        with self.assertRaises(ValueError):
+            self.frame.take(100, columns=['not_in'])
+
+    def test_frame_create_row_count(self):
+        """ Trivial Frame creation. """
+        frame = self.context.frame.import_csv(self.dataset, schema=self.schema)
+        self.assertEqual(frame.row_count, 3)
+        self.assertEqual(len(frame.take(3).data), 3)
```

Comment:
The take should try to take much more, else the test misses that take only finds 3 rows.
And, especially with only 3 rows, we should just compare the data and see that it is appropriate